swagger.yaml

```diff
@@ -0,0 +1,226 @@
+    get:
+      tags:
+        - tweets
+      description: "Find a tweet by tweet ID"
+      parameters:
+        - name: id
+          in: path
+          required: true
+          type: string
+
+      responses:
+        200:
+          description: Successful response
+          schema:
+            $ref: "#/definitions/tweetsData"
+        400:
+          description: Invalid request
+          schema:
+            $ref: "#/definitions/Error"
+        404:
+          description: tweet does not exist
+          schema:
+            $ref: "#/definitions/Error"
+        500:
+          description: Unexpected server error
+          schema:
+            $ref: "#/definitions/Error"
+
+    post:
+      tags:
+      - tweets
+      description: "post a new tweet"
+      parameters:
+        - name: body
+          in: body
+          description: "tweet will be created and posted"
+          required: true
+          schema:
+            $ref: "#/definitions/tweetsData"
+        - name: id
+          in: path
+          required: true
+          type: string
+
+
+      responses:
+        201:
+          description: Successfully posted
+          schema:
+            $ref: "#/definitions/tweetsData"
+        400:
+          description: Invalid syntax in request
+          schema:
+            $ref: "#/definitions/Error"
+        500:
+          description: Internal server error
+          schema:
+            $ref: "#/definitions/Error"
+
+    put:
+      tags:
+      - tweets
+      description: "update a tweet"
+      parameters:
+        - name: id
+          in: path
+          description: "update the tweet"
+          required: true
+          type: string
+
+        - name: body
+          in: body
+          description: "tweet content to be updated"
+          required: true
+          schema:
+            $ref: "#/definitions/tweetsData"
+
+      responses:
+        200:
+          description: Successful response
+          schema:
+            $ref: "#/definitions/tweetsData"
+        400:
+          description: Invalid request
+          schema:
+            $ref: "#/definitions/Error"
+        404:
+          description: page not found
+          schema:
+            $ref: "#/definitions/Error"
+        500:
+          description: Internal server error
+          schema:
+            $ref: "#/definitions/Error"
+
+    delete:
+      tags:
+        - tweets
+      description: "Delete a tweet"
+      parameters:
+        - name: id
+          in: path
+          description: "delete the tweet"
+          required: true
+          type: string
+
+      responses:
+        204:
+          description: Successfully removed
+          schema:
+            $ref: "#/definitions/tweetsData"
+        404:
+          description: tweet does not exist
+          schema:
+            $ref: "#/definitions/Error"
+        500:
+          description: Internal server error
+          schema:
+            $ref: "#/definitions/Error"
+
+
```

Comment:
Extra blank lines. There are also some in other places, can you remove them as well?